$OpenBSD: patch-src_gui_image_qpixmap_x11_cpp,v 1.1 2013/10/24 08:39:14 sthen Exp $

Partially revert fix for CVE-2013-0254 which does not work with our
privilege-separated X.

--- src/gui/image/qpixmap_x11.cpp.orig	Thu Oct 10 16:46:12 2013
+++ src/gui/image/qpixmap_x11.cpp	Thu Oct 10 16:34:39 2013
@@ -193,7 +193,7 @@ static bool qt_create_mitshm_buffer(const QPaintDevice
     bool ok;
     xshminfo.shmid = shmget(IPC_PRIVATE,
                              xshmimg->bytes_per_line * xshmimg->height,
-                             IPC_CREAT | 0700);
+                             IPC_CREAT | 0777);
     ok = xshminfo.shmid != -1;
     if (ok) {
         xshmimg->data = (char*)shmat(xshminfo.shmid, 0, 0);
